src/components/Guilds/Header/NetworkButton.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { observer } from 'mobx-react';
+import { isChainIdSupported } from 'provider/connectors';
+import NetworkModal from 'components/NetworkModal';
+import { useContext } from '../../../contexts';
+import { toCamelCaseString } from '../../../utils';
+import { Button, IconButton } from '../common/Button';
+
+import arbitrumIcon from '../../../assets/images/arbitrum.png';
+import ethereumIcon from '../../../assets/images/ethereum.svg';
+import xdaiIcon from '../../../assets/images/xdai.svg';
+
+const iconsByChain = {
+  1: ethereumIcon,
+  4: ethereumIcon,
+  100: xdaiIcon,
+  42161: arbitrumIcon,
+  421611: arbitrumIcon,
+  1337: ethereumIcon,
+};

```

Comment:
Can't we place this in constants?

Reply:
This was in constants, but I think we moved this here as its better to only import it when we use it. (These icons are somewhat large.)